config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "pc22",
+    "password": "Lilac@PC22#",
+    "database": "mealmedatabase",
+    "host": "localhost",
+    "dialect": "postgres"
+  },
+  "test": {
+    "username": "pc22",
+    "password": "Lilac@PC22#",
+    "database": "mealmedatabase",
+    "host": "localhost",
+    "dialect": "postgres"
+  },
+  "production": {
+    "username": "pc22",
+    "password": "Lilac@PC22#",
+    "database": "mealmedatabase",
+    "host": "localhost",
+    "dialect": "postgres"
+  }
+}
```

Comment:
Good to see this config. This is the standard way of setting up any environment. 
It was missing in the code sent by @Sharooq-KN 
But now that it's here, it's good.

Reply:
But, the better usage is to convert this file `config.json` from **JSON** to **JS**

And, export the entire object.

Reply:
ok

Reply:
Also, after converting to **JS** populate the configuration directives using the ENV variables. 

For eg:
```javascript
"username": process.env.DB_USER
```